e2e-tests/trade-finance-manager/cypress/integration/journeys/case/amendments/show-amendment-button.spec.js

```diff
@@ -100,52 +100,7 @@ context('Amendments page', () => {
     });
   });
 
-  describe('MIN - not confirmed', () => {
```

Comment:
note: I've removed this one because it's not relevant anymore. once a MIA changes to MIN, the status changes to `Confirmed` for a MIN deal. this is a new feature added directly from `main` branch